modules/apps/data-engine/data-engine-taglib/src/main/resources/META-INF/resources/data_layout_builder/js/AppContext.es.js

```diff
@@ -177,17 +177,29 @@ const editFocusedCustomObjectField = ({
 	};
 };
 
+/**
+ * Get unformatted definition field
+ * @param {object} dataDefinition
+ * @param {string} fieldName
+ */
+const getUnformattedDefinitionField = (dataDefinition, {fieldName}) => {
+	return getDataDefinitionField(dataDefinition, fieldName);
+};
+
+/**
+ * Get formatted definition field
+ * @param {object} dataLayoutBuilder
+ * @param {string} fieldName
+ */
+const getFormattedDefinitionField = (dataLayoutBuilder, field) => {
```

Comment:
This second parameter, could be fieldName, as written in the jsDOC at line 192?

Reply:
Actually, i have seen that you are passing this field (from ddm) entirely, so, this way you could keep the "field" as second parameter name, just update the param type and name on jsDOC, line 192

Reply:
Yeah, I'll do it. Thanks!